src/DynamicObjects.jl

```diff
@@ -34,13 +38,9 @@ This dynamic type can be used in function definitions as
 area(what::Rectangle) = what.height * what.width
 ```
 """
-get_sname(name::Symbol) = QuoteNode(name)
-get_sname(name::Expr) = QuoteNode(name.args[1])
-ename_and_ebase(name::Symbol, default) = esc(name), esc(default)
-ename_and_ebase(name::Expr, default) = esc.(name.args)
 macro dynamic_object(name, args...)
     sname = get_sname(name)
-    ename, ebase = ename_and_ebase(name, :DynamicObject)
+    ename, ebase = ename_and_ebase(name, DynamicObject)
```

Comment:
Hm, that sounds as if it does the right thing :) I had this locally:
```
ename, ebase = ename_and_ebase(name, :(DynamicObjects.AbstractDynamicObject))
```

Reply:
Your line still requires `DynamicObjects` to be in the user's module's namespace, but it might have been e.g. `import DynamicObjects as DO`. The right thing to do is to explicitly refer to the _object_ in the current (DynamicObjects.jl) namespace, not to a symbol.

Reply:
Makes sense!